lore/learningobjects/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Course(models.Model):
```

Comment:
so this is not part of lore? if they are sep apps, then, being not part of lore, having them in repo_root makes sense. maybe we can change 'lore' to 'lore_core' or something less confusing in this architecture if learningobjects is a part of lore.